tests/data/RunInfo.xml

```diff
@@ -7,7 +7,7 @@
     <Reads>
       <Read Number="1" NumCycles="151" IsIndexedRead="N" />
       <Read Number="2" NumCycles="8" IsIndexedRead="Y" />
-      <Read Number="3" NumCycles="151" IsIndexedRead="N" />
+      <Read Number="3" NumCycles="151" IsIndexedRead="Y" />

```

Comment:
151 cycles cannot be index reads. Why it was changed to "Y"?

Reply:
This was to make a different case, to improve the test coverage, but yeah, what you say makes sense. Would changing NumCycles to something other than 151 make sense?

Reply:
Index cycles are usually 6-17nt. You may change it within that range If it is not too complicated for the other parts

Reply:
Great, fixed it.